### PR TITLE
Docs: Fix api url examples, expand notes

### PR DESCRIPTION
**Pull request by tpdorsey**
### Summary

Fix API URL examples in docs.

Add some notes about configuring API URL where it's needed (deployments and agents).

Add a specific section in the settings docs.

### Checklist

This pull request is:

- [x] A documentation / typographical error fix
	- No tests or issue needed
- [ ] A short code fix
	- Please reference the related issue by including "closes `<link to issue>`" in this Pull Request's summary section. 
        - If no issue exists, please create a bug report issue 
	- Please include tests. One-line fixes without tests will not be accepted unless it's related to the documentation only.
- [ ] A new feature implementation
	- Please reference the related issue by including "closes `<link to issue>`" in this Pull Request's summary section. 
        - If no issue exists, please create a feature enhancement issue 
	- Please include tests
    - Please make sure that your QA steps are both thorough and easy to reproduce by somebody with limited knowledge of the feature that you are submitting

**Happy engineering!**

### CODE REVIEW

File: docs/concepts/settings.md [hunk 1]
```diff
@@ -17,6 +17,73 @@ Settings have keys that match environment variables that can be used to override
...
+```
+
+!!! tip "`PREFECT_API_URL` setting for agents"
+    When using [work queues and agents](/concepts/work-queues/) that can create flow runs for deployments in remote environments,  [`PREFECT_API_URL`](/concepts/settings/) must be set for the environment in which your agent is running. 
+
+    If you want the agent to communicate with Prefect Cloud or a Prefect Orion API server instance from a remote execution environment such as a VM or Docker container, you must configure `PREFECT_API_URL` in that environment.
+
+### PREFECT_API_KEY
+
+The `PREFECT_API_KEY` value specifies the [API key](/ui/cloud-getting-started/#create-an-api-key) value for interacting with a Prefect Cloud workspace.
```
Comment (anna-geller) [inline docs/concepts/settings.md:46]:
```suggestion
The `PREFECT_API_KEY` value specifies the [API key](/ui/cloud-getting-started/#create-an-api-key) used to authenticate with your Prefect Cloud workspace.
```

File: docs/concepts/settings.md [hunk 2]
```diff
@@ -17,6 +17,73 @@ Settings have keys that match environment variables that can be used to override
...
+
+```bash
+PREFECT_API_KEY="[API-KEY]"
```
Comment (anna-geller) [inline docs/concepts/settings.md:49]:
```suggestion
prefect config set PREFECT_API_KEY=your_api_key
```

Quotation marks are not strictly needed

File: docs/concepts/settings.md [hunk 3]
```diff
@@ -17,6 +17,73 @@ Settings have keys that match environment variables that can be used to override
...
+```
+
+### PREFECT_HOME
+
+The `PREFECT_HOME` value specifies the local Prefect directory for configuration files, profiles, and the location of the default [Prefect SQLite database](/concepts/database/).
+
+```bash
+PREFECT_HOME='~/.prefect'
```
Comment (anna-geller) [inline docs/concepts/settings.md:57]:
```suggestion
prefect config set PREFECT_HOME='~/.prefect'
```

File: docs/concepts/settings.md [hunk 4]
```diff
@@ -17,6 +17,73 @@ Settings have keys that match environment variables that can be used to override
...
+```
+
+### PREFECT_LOCAL_STORAGE_PATH
+
+The `PREFECT_LOCAL_STORAGE_PATH` value specifies the default location of local storage for flow runs.
+
+```bash
+PREFECT_LOCAL_STORAGE_PATH='${PREFECT_HOME}/storage'
```
Comment (anna-geller) [inline docs/concepts/settings.md:65]:
```suggestion
prefect config set PREFECT_LOCAL_STORAGE_PATH='${PREFECT_HOME}/storage'
```

Comment (anna-geller) [reply docs/concepts/settings.md:65]: same for all below

**Review by anna-geller — APPROVED** [review]
added minor comments but LGTM, thanks a lot

File: docs/concepts/settings.md (hunk 2 above)
Comment (tpdorsey) [reply docs/concepts/settings.md:49]: I'm concerned that showing the `config set` command confuses with simply showing the setting and value.

**Pull request merged**